# commands sent in json format
string msg
---
uint8 SUCCESS=0
uint8 FAILURE=1
uint8 result
string error_str
